src/Pororoca.Desktop/Styles/Accents/Light2.xaml

```diff
@@ -0,0 +1,56 @@
+<ResourceDictionary
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Pororoca.Desktop">
+  <ResourceDictionary.ThemeDictionaries>
+    <ResourceDictionary x:Key="{x:Static local:PororocaThemeManager.Light2}">
+      <!-- System colours overrides below -->
+      <Color x:Key="SystemAccentColorDark3">#4e6006</Color>
+      <Color x:Key="SystemAccentColorDark2">#759009</Color>
+      <Color x:Key="SystemAccentColorDark1">#9cc00c</Color>
+      <Color x:Key="SystemAccentColor">#c4f00f</Color>
+      <Color x:Key="SystemAccentColorLight1">#cff33f</Color>
+      <Color x:Key="SystemAccentColorLight2">#dbf66f</Color>
+      <Color x:Key="SystemAccentColorLight3">#e7f99f</Color>
+      <Color x:Key="SystemErrorTextColor">Red</Color>
+      <!-- Custom colours below -->
+      <Color x:Key="InputBackgroundValidationProblemColor">#99fecada</Color>
+      <Color x:Key="SystemInvertedAccentColor">#a90a4c</Color>
+      <Color x:Key="SystemInvertedAccentColorLight1">#b50a51</Color>
+      <Color x:Key="SystemInvertedAccentColorLight2">#c10b57</Color>
+      <Color x:Key="MainWindowMenuColor">#fcfcfc</Color>
+      <Color x:Key="MainWindowBackgroundColor">#fafafa</Color>
+      <Color x:Key="MainWindowDarkerBackgroundColor">#eaeaeb</Color>
+      <Color x:Key="MainWindowSidebarColor">#eaeaeb</Color>
+      <Color x:Key="MainWindowSeparatorsColor">#eaeaeb</Color>
+      <Color x:Key="CurrentEnvironmentLabelColor">#a90a4c</Color>
+      <Color x:Key="ListBoxBackgroundColor">#eaeaeb</Color>
+      <Color x:Key="ListBoxItemSelectedBackgroundColor">#fcfcfc</Color>
+      <Color x:Key="DragAndDropItemDraggingColor">Black</Color>
+
+      <Thickness x:Key="ButtonSecondaryBorderThickness">0</Thickness>
+      <Color x:Key="ButtonSecondaryBorderBrush">Transparent</Color>
+      <Color x:Key="ButtonSecondaryHighlightBorderBrush">Transparent</Color>
+      <Color x:Key="ButtonSecondaryBackgroundColor">#eaeaeb</Color>
+      <Color x:Key="ButtonSecondaryHighlightBackgroundColor">#cff33f</Color>
+
+      <Color x:Key="WebSocketConnectedColor">#a90a4c</Color>
+      <Color x:Key="WebSocketMessageFromClientPrimaryColor">#780736</Color>
+      <Color x:Key="WebSocketMessageFromServerPrimaryColor">Green</Color>
+      <Color x:Key="DefaultIconsColor">Black</Color>
```

Comment:
I think that the icon color would look better matching the text color (#777777):

Before:

![Captura de tela 2024-08-26 110238](https://github.com/user-attachments/assets/f7b5a24b-5ad0-48c6-90eb-9dbfa930deda)

![Captura de tela 2024-08-26 110845](https://github.com/user-attachments/assets/c49a8a62-b43a-4536-9509-3c29a6c23ab6)

After:

![Captura de tela 2024-08-26 110148](https://github.com/user-attachments/assets/4516a962-ecc6-4a9e-b7d1-22be5870f3ec)

![Captura de tela 2024-08-26 110935](https://github.com/user-attachments/assets/adb627b2-bce8-4ac9-a7dc-3b31bd122f53)